sormas-rest/src/main/java/de/symeda/sormas/rest/ExternalVisitsResource.java

```diff
@@ -30,6 +35,23 @@ public Boolean isValidPersonUuid(@PathParam("personUuid") String personUuid) {
 		return FacadeProvider.getPersonFacade().isValidPersonUuid(personUuid);
 	}
 
+	@POST
+	@Path("/person/{personUuid}/status")
+	public boolean postSymptomJournalStatus(@PathParam("personUuid") String personUuid, Map<String, Object> body) {
+		try {
+			PersonDto person = FacadeProvider.getPersonFacade().getPersonByUuid(personUuid);
```

Comment:
usually it is desirable not to have any kind of logic in the resource layer - i would advise to expose a specific method in the facade